spec/models/students_classrooms_spec.rb

```diff
@@ -1,14 +1,64 @@
 require 'rails_helper'
 
-describe StudentsClassrooms, type: :model do
-
-  context "it listens to the coder" do
-    it "makes one student classroom and no students if told to" do
-      StudentsClassrooms.create!(classroom_id: 1, student_id: 1)
-      expect(StudentsClassrooms.all.count).to eq(1)
-      expect(User.where(role: 'student').count).to eq(0)
-      expect(Classroom.all.count).to eq(0)
-    end
-  end
+describe StudentsClassrooms, type: :model, redis: :true do
+	it { should belong_to(:student).class_name('User') }
+	it { should belong_to(:classroom).class_name("Classroom") }
```

Comment:
Is `.class_name("Classroom")` necessary here?

Reply:
yes since it's used in the code, the point is to have a test for each line/word of code written